lib/rubocop/cop/rspec/rails/minitest_assertions.rb

```diff
@@ -229,6 +252,45 @@ def assertion
               'be_empty'
             end
           end
+
+          MSG = 'Use `%<prefer>s`.'
+
+          ASSERTION_MATCHERS = constants(false).filter_map do |c|
```

Comment:
BasicAssertion.subclasses ? Or is it a railsism?

Reply:
No that's a ruby 3.1ism - I've added a note over in #1790 about switching when 3.1 becomes the min.